### DISCUSSION TITLE
is this the recommended way to manage NATS cluster?

### DISCUSSION THREAD

**Discussion by piengeng** [nats-io/nsc#646] [Q&A]
is there any new documentation on how to properly manage NATS cluster, or should all user(s) management should be done in the *.conf file, and perform reload via cli?

**Comment by aricart** [comment]
nsc only concerns itself with delegate authentication - it simply creates a chain of trust for operator/account/users. You'll still need a server configuration to configure your cluster.

See https://docs.nats.io/running-a-nats-service/configuration
